TelCo.ColorCoder/ColorConstant.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Text;
+
+namespace TelCo.ColorCoder
+{ 
+    public static class ColorConstant

```

Comment:
do not name class assuming these colors are constant, if there is a color to be introduced later, then the name does not hold